pennylane/about.py

```diff
@@ -12,34 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 r"""
-About
+about
 =====
 
-**Module name:** :mod:`pennylane.about`
+**Module name**: pennylane.about
 
 .. currentmodule:: pennylane.about
 
-A simple module to display all the details of the `pennylane` installation,
+This module contains the :func:`about` function to display all the details of the `pennylane` installation,
 e.g., OS, version, `Numpy` and `Scipy` versions, installation method.
```

Comment:
This module contains the :func:`about` function which can be used to display all the details of PennyLane installation,